### PR TITLE
feat: Support `OFFSET` for `PostgreSQL`

### LINKED ISSUE #68: Support `OFFSET` for PostgreSQL
Depends on #67

### PR DESCRIPTION
**Pull request by viladimiru**
Closes #68

### CODE REVIEW

File: src/parsing/parsers/postgresql/jison/select/limit_clause.jison [hunk 1]
```diff
@@ -21,7 +21,6 @@ OptionalLimitClause
 
 LimitClause
  : 'LIMIT' UnsignedNumericLiteral
- | 'LIMIT' 'VARIABLE_REFERENCE'
```
Comment (viladimiru) [inline src/parsing/parsers/postgresql/jison/select/limit_clause.jison:24]: PostgreSQL didn't support `${variable}` conditions

File: src/parsing/parsers/postgresql/jison/select/offset_clause.test.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    "afterCursor": "",
+    "noErrors": true,
+    "containsKeywords": ["SELECT"],
+    "expectedResult": {
+      "lowerCase": false
+    }
+  },
+  {
+    "namePrefix": "should not throw errors",
+    "beforeCursor": "SELECT * FROM testTable LIMIT 1 OFFSET 12; ",
```
Comment (viladimiru) [inline src/parsing/parsers/postgresql/jison/select/offset_clause.test.json:24]: Created an issue to support queries without ordering `LIMIT` and `OFFSET` https://github.com/gravity-ui/sql-autocomplete-parsers/issues/72

**Review by NikitaShkaruba — CHANGES_REQUESTED** [review]
Great! Let's fix the copyright and we can merge it.

File: src/parsing/parsers/postgresql/jison/select/offset_clause.test.json (hunk 2 above)
Comment (NikitaShkaruba) [reply src/parsing/parsers/postgresql/jison/select/offset_clause.test.json:24]: Yeah, we need more tests in the future!

**Review by NikitaShkaruba — APPROVED** [review]
Great, let's merge it!

**Pull request merged**